src/__test__/utils/sendWork.test.js

```diff
@@ -64,12 +76,12 @@ describe('sendWork unit tests', () => {
   });
 
   it('Sends work to the backend when called and returns valid response.', async () => {
-    const response = await sendWork(
-      experimentId, timeout, body,
+    const response = await seekFromAPI(
+      experimentId, body, timeout, 'badbeef',
     );
 
     expect(socketConnectionMocks.mockEmit).toHaveBeenCalledWith('WorkRequest', {
-      uuid: 'my-random-uuid',
+      ETag: 'badbeef',
```

Comment:
why `badbeef` when everything in this test is supposed to work without errors?

Reply:
It's a common hexadecimal value used for representing a garbage value. I understand the confusion, I'll just swap it with something else, like `facefeed`.

Reply:
fair enough